kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkTaskConfig.java

```diff
@@ -112,6 +112,15 @@
           + " and enable timestamp partitioning for each table. Leave this configuration blank,"
           + " to enable ingestion time partitioning for each table.";
 
+  public static final String BIGQUERY_PARTITION_EXPIRATION_CONFIG = "partitionExpirationMs";
+  private static final ConfigDef.Type BIGQUERY_PARTITION_EXPIRATION_TYPE = ConfigDef.Type.LONG;
+  private static final String BIGQUERY_PARTITION_EXPIRATION_DEFAULT = null;
+  private static final ConfigDef.Importance BIGQUERY_PARTITION_EXPIRATION_IMPORTANCE = ConfigDef.Importance.LOW;
+  private static final String BIGQUERY_PARTITION_EXPIRATION_DOC =
+          "The amount of time, in milliseconds, after which partitions should be deleted from the table. If this " +
+                  "field is set, all data in partitions older than the specified partition expiration time will be " +
+                  "permanently deleted.";
```

Comment:
"...all data in partitions **in this table** older than..."?

Reply:
Hmmm... I think we might want to be more explicit. I tried to spell it out plainly with the recently-introduced `timePartitioningType` property's docs: https://github.com/confluentinc/kafka-connect-bigquery/blob/03480f57fdd917454a237b407d4fe32e53879034/kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java#L310-L312

Maybe instead of "deleted from the table", we could say "deleted from tables that this connector creates", and append the "Existing tables will not be altered to use this expiration time." note to the end. How does that sound to you?

Reply:
I think I get it now. That still leaves a bit of ambiguity around the "...all data in partitions in this table older than..." part. How about this?
> The amount of time, in milliseconds, after which partitions should be deleted from the tables this 
> connector creates. If this field is set, all data in partitions in this connector's tables that are 
> older than the specified partition expiration time will be permanently deleted. 
> Existing tables will not be altered to use this partition expiration time.

Reply:
That sounds great, yeah!